doc/api/vm.md

```diff
@@ -479,8 +479,17 @@ within which it can operate. The process of creating the V8 Context and
 associating it with the `sandbox` object is what this document refers to as
 "contextifying" the `sandbox`.
 
+## Usage with `--abort-on-uncaught-exception`
+
+When Node.js is run with the `--abort-on-uncaught-exception` command line flag,
+exceptions thrown from within a VM context are treated as uncaught, and cause
+the process to abort accordingly.
+[`process.setUncaughtExceptionCaptureCallback()`][] can be used to prevent the
+process from aborting.
+
 [`Error`]: errors.html#errors_class_error
 [`eval()`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval
+[`process.setUncaughtExceptionCaptureCallback()`]: process.html#process_process_setuncaughtexceptioncapturecallback_fn
```

Comment:
Is this documented yet? I cannot find it in the last nightly docs:

https://nodejs.org/download/nightly/v10.0.0-nightly201711280fb1e07689/docs/api/process.html#process_process_setuncaughtexceptioncapturecallback_fn

Reply:
It landed this morning so will need another night to pass :)

Reply:
Sorry)